src/main/java/org/opentripplanner/client/model/VertexType.java

```diff
@@ -0,0 +1,9 @@
+package org.opentripplanner.client.model;
+
+public enum VertexType {
```

Comment:
The vertex type is a very questionable aspect of the current API and has many strange values carried over from OTP1. What do you need it for?

Reply:
All right. 
We used it, to differ between different kinds of POIs. 
But we don't need this actually. We can use the different properties stop, vehicleRentalStation, rentalVehicle, rentalVehicle instead.
 I can remove the vertexType

Reply:
I removed the vertexType